doc/api/unicode.md

```diff
@@ -0,0 +1,55 @@
+# icu
+
+    Stability: 1 - Experimental
+
+The `unicode` module provides an interface to Unicode and Internationalization
+functionality provided by the ICU4C library.
+
+```js
+const unicode = require('unicode');
+```
+
+## unicode.transcode(buf, from_enc, to_enc)
+
+* `buf` {Buffer} A `Buffer` instance
+* `from_enc` {string} The current encoding
+* `to_enc` {string} The target encoding
+
+Re-encodes the given `Buffer` from one character encoding to another. Returns
+a new `Buffer` instance.
+
+## unicode.codePointAt(buf, pos, encoding)
+
+* `buf` {Buffer} A `Buffer` instance
+* `pos` {integer} The offset position
```

Comment:
I would add `in bytes` or something similar here to make clear that this is not measured in characters.